### ISSUE TITLE
Project Links Required

### ISSUE THREAD

**Issue by RamchandraWarang9822** [xitanggg/open-resume#101]
### Most of the project can be previewed by an URL 
- Having an option to add preview  or source code links for Software projects would be really helpful.

![image](https://github.com/xitanggg/open-resume/assets/92023869/9a3200ba-a976-40ce-ac31-674d7625c289)

**Comment by Sahil-Sharma-23** [comment]
Add your requested feature @RamchandraWarang9822

**Comment by devAlisha** [comment]
After downloading Project title and link both are not visible

**Comment by RamchandraWarang9822** [comment]
> After downloading Project title and link both are not visible @devAlisha 

Can you add screenshots related to same ?

**Comment by devAlisha** [comment]
### Steps to reproduce:

1. Add project title, desc and link
2. ![image](https://github.com/xitanggg/open-resume/assets/134960860/9a82a4ce-4c32-41b8-96a4-fced31ba8695)
3. Download the Resume
4. ![image](https://github.com/xitanggg/open-resume/assets/134960860/0ce995d1-c51d-4ad5-b713-eff98a558b60)
5. **In the download resume pdf, the Project title and Project Link both are not visible.**

**Comment by RamchandraWarang9822** [comment]
@xitanggg 
Should we continue working on the feature ? 

It is a necessary feature for Project links to be embedded in Resume for preview 

Please confirm so that we can continue contributing.

**Comment by Sahil-Sharma-23** [comment]
> ### Steps to reproduce:
> 1. Add project title, desc and link
> 2. ![image](https://private-user-images.githubusercontent.com/134960860/347695299-9a82a4ce-4c32-41b8-96a4-fced31ba8695.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE1OTM2MTgsIm5iZiI6MTcyMTU5MzMxOCwicGF0aCI6Ii8xMzQ5NjA4NjAvMzQ3Njk1Mjk5LTlhODJhNGNlLTRjMzItNDFiOC05NmE0LWZjZWQzMWJhODY5NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyMVQyMDIxNThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zOGE4ZTYyZTc0NTQyOTA3MTgxNWM5YTRhMzQwZDNkNDEzOTY1NmY0NDIzZDJmYWVmNmIwZDg0MDIzNGQ2YjFhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.quRyjT0-q1cBFlr3tW6662qALYOu3aHAFdz5Qi9DtYU)
> 3. Download the Resume
> 4. ![image](https://private-user-images.githubusercontent.com/134960860/347695739-0ce995d1-c51d-4ad5-b713-eff98a558b60.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE1OTM2MTgsIm5iZiI6MTcyMTU5MzMxOCwicGF0aCI6Ii8xMzQ5NjA4NjAvMzQ3Njk1NzM5LTBjZTk5NWQxLWM1MWQtNGFkNS1iNzEzLWVmZjk4YTU1OGI2MC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyMVQyMDIxNThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MmY0YjQxOTIyZGI3YmUxNDEzNTljNTkyNjA2YTgwZTM0YmU4Njc4ODEzNzc4YzVmMjMzMDM5YzJlN2U3OTU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.32fSPf2hmc14TMqB4Mvj4w9jvkOPIUk4htQ6VO-dHaw)
> 5. **In the downloaded resume pdf, the Project title and Project Link both are not visible.**

Thanks for letting me know @devAlisha 
I'll look into the issue.

**Comment by Sahil-Sharma-23** [comment]
> @xitanggg Should we continue working on the feature ?
> 
> It is a necessary feature for Project links to be embedded in Resume for preview
> 
> Please confirm so that we can continue contributing.

Yes, an update from @xitanggg would be good. So I'll immediately start working on the fix.
